src/pages/Swap/Swap.tsx

```diff
@@ -41,6 +58,99 @@ export const Swap: React.FC<SwapProp> = ({
     }
   };
 
+  const amountIsPositive = (x: any): boolean => {
+    if (!Number.isNaN(x) && Number(x) > 0) {
+      return true;
+    }
+
+    return false;
+  };
+
+  const handleSendAmountChange = async (
```

Comment:
you should clean the error with `setPreviewValueError(null)` to be sure if the user enters again different number int he input, does not show old error not relevant anymore

Reply:
here and also in `handleReceiveAmountChange`

Reply:
fixed this.